inc/saitunnel.h

```diff
@@ -824,18 +830,39 @@ typedef enum _sai_tunnel_term_table_entry_attr_t
      *
      * @type sai_ip_address_t
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @condition SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_TYPE == SAI_TUNNEL_TERM_TABLE_ENTRY_TYPE_P2P or SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_TYPE == SAI_TUNNEL_TERM_TABLE_ENTRY_TYPE_P2MP
      */
     SAI_TUNNEL_TERM_TABLE_ENTRY_ATTR_DST_IP,
 
+    /**
+     * @brief Tunnel termination IP address mask
+     *
+     * @type sai_ip_address_t
+     * @flags CREATE_ONLY
+     * @default 0.0.0.0
```

Comment:
default to be 255.255.255.255 to enable backward compatiblity?

Reply:
since it is not Mandatory-on Create it will be backward compatible.  But I feel default we can remove as based on v4/v6 it's value will be different . When this attribute is user is always expected to pass value.

Reply:
Ack. Removed default.

Reply:
what does this 0.0.0.0 mean here? I can see the default is here.

Reply:
Sorry I hit reply on comment too soon. I have edited my prev comment. Please let me know if your question is not answered by the updated reply.